panel/src/components/Forms/Blocks/Types/Text.vue

```diff
@@ -17,10 +15,23 @@
  * @internal
  */
 export default {
-	props: {
-		endpoints: Object
-	},
 	computed: {
+		component() {
+			const type = this.textField.type;
+			if (type === "writer") {
+				return "k-writer";
+			}
```

Comment:
We could simplify this further and remove this if statement. Then the code would check for `k-writer-input` which does not exist and still fall back to `k-writer`.

Reply:
I thought about it but wanted to avoid any extra checking 👍